src/components/daily/bolustooltip/BolusTooltip.js

```diff
@@ -172,10 +173,48 @@ class BolusTooltip extends React.Component {
     return extendedLine;
   }
 
+  getFormatedInputTime(bolus, timePrefs) {
+    const time = formatLocalizedFromUTC(bolus.inputTime, timePrefs, getHourMinuteFormat());
+    return time;
+  }
+
+  getBolusTypeLine(bolusType) {
+    return bolusType && (
```

Comment:
I know it works, but I do not like this method of returning.
it's confusing I think, a test should not be use to directly return an expression, based on Javascript poor design.

Suggestion using lodash (to change a little bit)
```suggestion
    return _.isString(bolusType) ? (
```

It's the same for the others functions below, no need to repeat.

Reply:
yep but that would mean that I change all the similar patterns which is not the scope of this PR.